rocky/reports/report_types/helpers.py

```diff
@@ -59,10 +59,23 @@ def get_report_types_for_oois(oois: list[str]) -> set[type[BaseReport]]:
     """
     Get all report types that can be generated for a given list of OOIs
     """
-
     return {report for ooi_pk in oois for report in get_report_types_for_ooi(ooi_pk)}
 
 
+def get_report_types_for_ooi_type(ooi_type: str) -> list[type[BaseReport]]:
```

Comment:
Any reason why this function returns a list, and `get_report_types_for_ooi_types` a set (or vice versa)?